kubernetes/testing_vars.yml

```diff
@@ -31,8 +31,8 @@ MEMCACHED_VERSION: "1.5.19"
 POSTGRES_VERSION: "11.5"
 REDIS_VERSION: "5.0.6"
 
-OPENDEBATES_IMAGE: docker.io/dpoirier/opendebates
-OPENDEBATES_VERSION: "20191002081650"
+OPENDEBATES_IMAGE: docker.io/caktus/opendebates
+OPENDEBATES_VERSION: "20191008110643"
```

Comment:
I find these dates hard to parse and generally prefer commit SHA tagged images, but that's also an easy change to make later if we want to.

Reply:
Not that I _can_ parse commit SHAs, but at least they track back to something tangible. 🙂